### PR TITLE
Fix register multisignature appending plus - Closes #470

### LINKED ISSUE #470: Command create transaction register multisignature fails
### Description

when we try to create transaction to register multisignature, it will fail with wrong hex.

### Steps to Reproduce
```
create transaction register multisignature account 24 2 215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bca 922fbfdd596fa78269bbcadc67ec2a1cc15fc929a19c462169568d7a3df1a1aa
{"error":"Could not create \"register multisignature account\" transaction: Argument must be a valid hex string."}
```
**Expected behavior:** [What you expect to happen]
you should be able to create transaction object.

**Actual behavior:** [What actually happens]
get the error
**Reproduces how often:** [What percentage of the time does it reproduce?]

### Versions
v1.0.0

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?
`create transaction register multisignature account` did not work with error `{"error":"Could not create \"register multisignature account\" transaction: Argument must be a valid hex string."}`.

`Lisk-js` was expecting keygroup array without `+`.

merge after #461 

### How did I fix it?
Deleting appending `+` on the lisk-commander.

### How to test it?
try command from example
```
create transaction register multisignature account 24 2 215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bca 922fbfdd596fa78269bbcadc67ec2a1cc15fc929a19c462169568d7a3df1a1aa
```

### Review checklist

* The PR solves #470 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/specs/commands/create_transaction_cast_votes.js [hunk 1]
```diff
@@ -74,8 +74,8 @@ describe('create transaction cast votes', () => {
 																			then.itShouldNotGetTheDataUsingTheUnvotesSource,
```
Comment (willclarktech) [inline test/specs/commands/create_transaction_cast_votes.js]: Do we not have any tests for transactions with votes and unvotes at the same time?

Comment (shuse2) [reply test/specs/commands/create_transaction_cast_votes.js]: It should.
https://github.com/LiskHQ/lisky/pull/471/commits/0a3a7ee1ce64db0388b143a3a6a9676492973a37#diff-3b2b6706fb38e00a91285dd3f43c8774L37

this one is the regarding test I think.

Comment (willclarktech) [reply test/specs/commands/create_transaction_cast_votes.js]: Right, but the name of that step should be changed now we're not prepending anything, no?

Comment (shuse2) [reply test/specs/commands/create_transaction_cast_votes.js]: yes that is true! Updated!

**Pull request merged**